src/Components/pages/Mission.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissions, getMission } from '../../Redux/missions/missions';
+import MissionRows from '../Missions/MissionRows';
+import '../../App.css';
+
+function Mission() {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.missions.missions);
+
+  const handleJoinMission = (id) => {
+    dispatch(getMission(id));
+  };
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, [dispatch]);
+
```

Comment:
- [ ]  [Optional] it would be nice to still see the missions I reserved after I navigate out of mission page and back, the `fetcMission` dispatch could be called only on initial render

Reply:
Thanks for the notice @VICTOR-CHUKWU, I will work on that.